lib/src/widgets/wolt_navigation_toolbar.dart

```diff
@@ -0,0 +1,190 @@
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+
+/// [WoltNavigationToolbar] is a modified version of the [NavigationToolbar] widget in the
+/// Flutter SDK. The primary modification is in the [performLayout] method of the
+/// [_ToolbarLayout] class.
+///
+/// This modification ensures a symmetric layout of widgets in the absence of either
+/// a leading or trailing widget, which makes for a visually balanced and
+/// aesthetically pleasing user interface.
+///
+/// This widget lays out its children in a manner suitable for a toolbar and
+/// positions three widgets or groups of widgets along a horizontal axis that's
+/// sensible for an application's navigation bar such as in Material Design and in iOS.
+///
+/// The [leading] and [trailing] widgets occupy the edges of the widget with
+/// reasonable size constraints while the [middle] widget occupies the remaining
+/// space in either a center aligned or start aligned fashion.
+///
+/// Modifications in [performLayout] ensure that if there is no leading widget and
+/// there is a trailing widget, then the width of the trailing widget is used as the
+/// width of the leading widget. Similarly, if there is no trailing widget and
+/// there is a leading widget, then the width of the leading widget is used as the
+/// width of the trailing widget.
+class WoltNavigationToolbar extends StatelessWidget {
```

Comment:
@ulusoyca 
Is this any different than https://github.com/TahaTesser/flutter/blob/master/packages/flutter/lib/src/widgets/navigation_toolbar.dart?

if isn't different or customized in any way, we should use `NavigationToolbar` directly instead of copying the source code and maintaining it.
If you need to call it `WoltNavigationToolbar` and not `NavigationToolbar`, we can return `NavigationToolbar`.

Reply:
@TahaTesser As mentioned in [here](https://github.com/woltapp/wolt_modal_sheet/pull/31/files/8120fc124b04baf000e075d023a3649015075e11#diff-fe2251d0ad25f10f7a8ec36ec804a6f0cb65936d5a35f3695e3f7385026cfa06R9), the NavigationToolBar doesn't center the middle in case the leading or trailing is missing. [Here is the change](https://github.com/woltapp/wolt_modal_sheet/pull/31/files/8120fc124b04baf000e075d023a3649015075e11#diff-fe2251d0ad25f10f7a8ec36ec804a6f0cb65936d5a35f3695e3f7385026cfa06R146)

Reply:
That makes sense if we've modifications.